python_modules/libraries/dagster-dbt/dagster_dbt/dbt_manifest_asset_selection.py

```diff
@@ -51,6 +52,7 @@ class DbtManifestAssetSelection(AssetSelection):
     dagster_dbt_translator: DagsterDbtTranslator
     exclude: str
     selector: str
+    project: Optional[DbtProject] = None
```

Comment:
Do we need a project here? Or could we just pass None to `select_unique_ids`? It looks like `select_unique_ids_from_manifest` doesn't use the project at all and since this class is `DbtManifestAssetSelection` we might not need it?

Reply:
Or do we use both the CLI and manifest for dbt Fusion? So we would reuse this class for dbt Fusion too?

Reply:
yep the goal is for us to re-use this class for dbt Fusion, which does need the project to be set (in order for it to know where to execute the cli command).